pages/workspace/tracking-plan/issue-reporter.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: 'guide'
+title: 'The Tracking Plan Issue Reporter'
```

Comment:
I'm wondering if people know what the "Issue Reporter" is, should we call it "Tracking Plan Validation", and maybe just "Validation" in the navbar instead? So it matches what this is called in the product